README.md

```diff
@@ -60,6 +60,13 @@ EOF
 networkattachmentdefinition.k8s.cni.cncf.io/macvlan-conf created
 ```
 
+## Collecting metrics with Prometheus
+Network attachment definition admission controller comes with following metrics.
+  1. No of pod's with k8s.v1.cni.cncf.io/networks annotations 
```

Comment:
Nit: No need to abbreviate, you can use number or `No.` for an abbreviation

Reply:
Fixed